Yoast/wordpress-seo
# Comment pagination notification conform to Yoast Alert standards

Moved Comment Pagination to Alert Center instead of on the "General" settings page.

**Testing**
1. Check the box on the `/wp-admin/options-discussion.php` page about comment pagination.
2. See if you have a new alert in your alert center
3. Disabling the box removes the alert again.

See #4650
